api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -126,6 +126,23 @@ public Workspace get(String ns, String firecloudName) {
         .collect(Collectors.toList());
   }
 
+  @Override
+  public List<WorkspaceResponse> getWorkspaces() {
+    return getWorkspacesAndPublicWorkspaces().stream()
+        .filter(
+            workspaceResponse ->
+                !(workspaceResponse.getAccessLevel() != WorkspaceAccessLevel.OWNER
```

Comment:
My reading of this line is that we'll include published workspaces in the "default" list only if the user is an OWNER on that workspace.

Should we also show them if the user is an EDITOR? It seems like the content team may be operating w/ a model where multiple folks are EDITORS, but only one individual is an OWNER. It may be odd to them if only the OWNER sees the workspace.

[Optional] Also – do we have any unit tests covering this logic? If it's not too much effort to add one, this logic feels just complicated enough to warrant some test coverage.

Reply:
yes i think we should add editors.  i don't love how this will not include people who are explicit READERS vs those in the all users group, but there isn't really a way to do that now that we are only fetching acls from firecloud.